app/controllers/users_controller.rb

```diff
@@ -10,7 +10,7 @@ def create
       }
       render json: { 'logged_in': true }
     else
-      render json: { errors: user.errors.full_messages }
+      render json: { errors: user.errors.full_messages }, status: :conflict
```

Comment:
#55 will have to be completed before I can continue with this. Any changes will break the build due to rubocop